### PR TITLE
KEYCLOAK-14417 Clarify the export/import documentation for the keyclo…

### PR DESCRIPTION
**Pull request by mposolda**
…ak.import mechanism

### CODE REVIEW

**Review by stianst — DISMISSED** [review]

File: server_admin/topics/export-import.adoc [hunk 1]
```diff
@@ -85,6 +85,11 @@ Examples:
 * -Dkeycloak.import=/tmp/realm1.json
 * -Dkeycloak.import=/tmp/realm1.json,/tmp/realm2.json
 
+NOTE: The mechanism based on `keycloak.import` cannot be used together with the other mechanism based on `keycloak.migration.X` properties.
+If it is used together, the `keycloak.import` will be ignored. The `keycloak.import` mechanism always ignore the realms, which already
```
Comment (andymunro) [inline server_admin/topics/export-import.adoc]: ignore should be ignores

File: server_admin/topics/export-import.adoc [hunk 2]
```diff
@@ -85,6 +85,11 @@ Examples:
...
+exists in {project_name}. Generally the `keycloak.import` mechanism can be convenient for some development purposes, but if more
```
Comment (andymunro) [inline server_admin/topics/export-import.adoc]: exists should be exist

File: server_admin/topics/export-import.adoc [hunk 3]
```diff
@@ -85,6 +85,11 @@ Examples:
...
+flexibility is needed, it is recommended to stick with the mechanism based on `keycloak.migration.X` properties.
```
Comment (andymunro) [inline server_admin/topics/export-import.adoc]: Change "it is recommended to" to "we recommend that you"

**Review by andymunro — CHANGES_REQUESTED** [review]
Just a few changes. thanks Marek.

File: server_admin/topics/export-import.adoc (hunk 1 above)
Comment (mposolda) [reply server_admin/topics/export-import.adoc]: I've addressed this.

File: server_admin/topics/export-import.adoc (hunk 2 above)
Comment (mposolda) [reply server_admin/topics/export-import.adoc]: I've addressed this.

File: server_admin/topics/export-import.adoc (hunk 3 above)
Comment (mposolda) [reply server_admin/topics/export-import.adoc]: I've addressed this.

**Comment by mposolda** [comment]
@andymunro Thanks! I've addressed your concerns. Do you please have a chance to re-review?

**Pull request merged**